### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by sandyhouse**
The initial release of PLSC (PaddlePaddle Large-Scale Classification) library, including the following features：
1. distributed fc-layer
2. distributed parameter pre-processor
3. pretrain-model download and upload

### CODE REVIEW

**Comment by gavin1332** [comment]
Description has to be added along with this PR.

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# 训练、评估和部署
+
+PaddlePaddle大规模分类提供了从训练、评估到预测部署的全流程解决方案。本文档介绍如何使用PaddlePaddle大规模分类库快速完成训练、评估和预测部署。
+
+## 数据准备
+
+我们假设用户数据集的组织结构如下：
```
Comment (danleifeng) [inline docs/usage.md]: 是否需要提供下我们这里用到的开源数据集链接？

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+
+### 快速入门
+
+* [安装说明](docs/installation.md)
+* [训练/评估/部署](docs/usage.md)
+
+### 基础功能
+
+* [API简介](docs/api_intro.md)
+* [自定义模型](docs/custom_modes.md)
```
Comment (danleifeng) [inline README.md]: 点击404，发现是docs/custom_modes.md 拼写错误

File: docs/export_for_infer.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# 预测模型导出
+
+通常，PaddlePaddle大规模分类库在训练过程中保存的模型只保存模型参数信息，
+而不包括预测模型结构。为了部署PLSC预测库，需要将预训练模型导出为预测模型。
+
+可以通过下面的代码将预训练模型导出为预测模型：
```
Comment (danleifeng) [inline docs/export_for_infer.md]: 能否解释一下：1. 预训练模型是哪一个，保存了什么 2. 预测模型是什么，保存了什么，需要什么。 3.导出预测模型用在哪里，怎么使用

File: docs/custom_models.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+用户自定义模型类需要继承自基类BaseModel，并实现build_network方法，实现用户的自定义模型。
+
+build_network方法的输入如下：
+* input: 输入图像数据
+* label: 图像类别
+* is_train: 表示训练阶段还是测试/预测阶段
+
+build_network方法返回用户自定义组网的输出变量，BaseModel类的get_output方法将调用该方法获取用户自定义组网的输出，并自动在其后添加分布式FC层。
```
Comment (danleifeng) [inline docs/custom_models.md]: get_output 方法如果用户感知不到可删掉，否则有点混淆

File: docs/usage.md (hunk 1 above)
Comment (sandyhouse) [reply docs/usage.md]: This section only gives a common structure of train datasets, but not a specific dataset.

File: README.md (hunk 2 above)
Comment (sandyhouse) [reply README.md]: done.

File: docs/export_for_infer.md (hunk 3 above)
Comment (sandyhouse) [reply docs/export_for_infer.md]: done

File: docs/custom_models.md (hunk 4 above)
Comment (sandyhouse) [reply docs/custom_models.md]: done

**Pull request merged**